lib/Service/FederatedUserService.php

```diff
@@ -192,7 +195,7 @@ public function __construct(
 		$this->interfaceService = $interfaceService;
 		$this->configService = $configService;
 
-		$this->cache = $cacheFactory->createDistributed(self::CACHE_SINGLE_CIRCLE);
+		$this->cache = $cacheFactory->createLocal(self::CACHE_SINGLE_CIRCLE);
```

Comment:
Why?

Reply:
cached data is static and have no interest being shared, therefor local should be better than distributed when distributed is not needed.

I might be in the wrong here, if local is not lighter than distributed I can revert this change

Reply:
Well it can be faster, but with a distributed cache the cache hit rate might be higher.

Reply:
ok, i'll revert to distributed